hack/local-up-cluster.sh

```diff
@@ -897,6 +918,8 @@ if [[ "${START_MODE}" != "kubeletonly" ]]; then
   start_apiserver
```

Comment:
actually, it looks like `kube::util::wait_for_url "http://${API_HOST_IP}:${API_SECURE_PORT}/healthz"` should switch to the secure port